### PR TITLE
Pract.05: Add 04-06 | Stefka Lyaskalieva

### PR DESCRIPTION
**Pull request by stefka04**
Stefka Lyaskalieva
2 4MI0600158

### CODE REVIEW

File: SI/Sem.05/Pract.05/Task04.cpp [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    }
+    for (int i = 0; i < Size/2; i++)
+    {
+        if (arr[i] != arr[Size - i - 1])
+        {
+            return false;
+        }
+    }
+    return true;
+}
```
Comment (GeorgiTerziev02) [inline SI/Sem.05/Pract.05/Task04.cpp:20]: This should be function that accepts int[] and length

File: SI/Sem.05/Pract.05/Task05.cpp [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+			else if (arr1[i] < arr2[j])
+			{
+				i++;
+			}
+			else if (arr1[i] > arr2[j])
+			{
+				j++;
+			}
+	} 
+}
```
Comment (GeorgiTerziev02) [inline SI/Sem.05/Pract.05/Task05.cpp:44]: Great

**Review by GeorgiTerziev02 — CHANGES_REQUESTED** [review]
Good job. Change the function on task04 as in the comment I left.

**Comment by Justsvetoslavov** [comment]
Cherry-picked 5 & 6

**Pull request closed without merging**